src/SprykerEco/Zed/AvalaraTax/Dependency/External/AvalaraTaxToAvalaraAvaTaxClientAdapter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTax\Dependency\External;
+
+use Avalara\AvaTaxClient;
+use SprykerEco\Zed\AvalaraTax\AvalaraTaxConfig;
+use stdClass;
+
+class AvalaraTaxToAvalaraAvaTaxClientAdapter implements AvalaraTaxToAvalaraTaxClientInterface
+{
+    /**
+     * @var \Avalara\AvaTaxClient
+     */
+    protected $avaTaxClient;
+
+    /**
+     * @param \SprykerEco\Zed\AvalaraTax\AvalaraTaxConfig $avalaraTaxConfig
+     */
+    public function __construct(AvalaraTaxConfig $avalaraTaxConfig)
+    {
+        $this->avaTaxClient = new AvaTaxClient(
+            $avalaraTaxConfig->getApplicationName(),
+            $avalaraTaxConfig->getApplicationVersion(),
+            $avalaraTaxConfig->getEnvironmentName(),
+            $avalaraTaxConfig->getMachineName()
+        );
+
+        $this
+            ->withLicenseKey($avalaraTaxConfig->getAccountId(), $avalaraTaxConfig->getLicenseKey())
+            ->withCatchExceptions(false);
+    }
+
+    /**
+     * @param int $accountId
+     * @param string $licenseKey
+     *
+     * @return $this
+     */
+    public function withLicenseKey(int $accountId, string $licenseKey)
```

Comment:
* If the methods are not used outside the class `withLicenseKey(), withCatchExceptions()` then remove them.